pipeline.py

```diff
@@ -143,6 +235,61 @@ def build_e2e_image(config: DevConfig) -> None:
         },
         inventory="inventories/e2e-inventory.yaml",
     )
+    sonar_build_image(
+        "e2e-amd64",
+        config,
+        args={
+            "registry": config.repo_url,
+            "base_image": release["golang-builder-image"],
+            "e2e_image": config.e2e_image,
+        },
+        inventory="inventories/e2e-inventory.yaml",
+    )
+
+    create_and_push_manifest(config, config.e2e_image)
+
+    if config.gh_run_id is not None and config.gh_run_id != "":
+        create_and_push_manifest(config, config.e2e_image, config.gh_run_id)
+
+
+def create_and_push_manifest(
+    config: DevConfig, image: str, tag: str = "latest"
+) -> None:
+    final_manifest = "{0}/{1}:{2}".format(config.repo_url, image, tag)
+    args = ["docker", "manifest", "rm", final_manifest]
+    args_str = " ".join(args)
+    print(f"removing existing manifest: {args_str}")
+    subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    args = [
+        "docker",
+        "manifest",
+        "create",
+        final_manifest,
+        "--amend",
+        final_manifest + "-amd64",
+        "--amend",
+        final_manifest + "-arm64",
+    ]
+    args_str = " ".join(args)
+    print(f"creating new manifest: {args_str}")
+    cp = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    if cp.returncode != 0:
+        raise Exception(cp.stderr)
+
+    args = ["docker", "manifest", "push", final_manifest]
+    args_str = " ".join(args)
+    print(f"pushing new manifest: {args_str}")
+    cp = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+
+    if cp.returncode != 0:
+        raise Exception(cp.stderr)
+
+
+# docker manifest rm $(REPO_URL)/$(OPERATOR_IMAGE):latest | true
```

Comment:
These steps should be part of the docstring for the above function, right?

Reply:
Oh yes, they describe what this method does. Do you think I should put them above the function, or delete them entirely?

Reply:
I think they should go above the function, maybe extend with a docstring, saying this does the equivalent of these commands.

Reply:
Yes, done!